docs/en/ingest-management/elastic-agent/configuration/inputs/input-winlog.asciidoc

```diff
@@ -0,0 +1,372 @@
+:input-type: winlog
+
+:vista_and_newer: This option is only available on operating systems +
+  supporting the Windows Event Log API (Microsoft Windows Vista and newer).
+
+[[winlog-input]]
+== Winlog input
+
+++++
+<titleabbrev>Winlog</titleabbrev>
+++++
+
+Configure the `winlog` input to read messages from Microsoft Windows
+Event Channels.
+
+To configure this input, specify a `name` of the Microsof Windows
+Event Channel to subscribe to.
+
+You can get a list of available event logs by running `Get-WinEvent
+-ListLog` in PowerShell.  Most channels are tied to specific event
+publishers. Here is an example showing how to list all channels using
+PowerShell.
```

Comment:
Not sure we have had it in the old docs, but using 'Event Viewer' allows users to inspect actual channels before configuring them. Maybe nice to have for "newcomers" that don't have much experience with windows event logs.